MvvmCross.Android.Support/V7.RecyclerView/MvxRecyclerAdapter.cs

```diff
@@ -215,54 +191,47 @@ protected virtual int GetItemsSourcePosition(int viewPosition)
 
         protected virtual void SetItemsSource(IEnumerable value)
         {
+            if (Looper.MainLooper != Looper.MyLooper())
+                Mvx.IoCProvider.Resolve<IMvxLog>().Error("ItemsSource property set on a worker thread. This leads to crash in the RecyclerView. It must be set only from the main thread.");
```

Comment:
Use MvxLog.Instance.

Reply:
MvxLog is internal and not available is this assembly.

Reply:
https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross.Android.Support/V7.RecyclerView/MvxAndroidLog.cs#L11

Reply:
Its not MvxLog, it's MvxAndroidLog.